pkg/log/log.go

```diff
@@ -16,7 +16,7 @@
 // Istio components.
 //
 // Istio's logging subsystem is built on top of the [Zap](https://godoc.org/go.uber.org/zap) package.
-// High performance scenarios should use the Error, Warn, Info, and Debug methods. Lower perf
+// High performance scenarios should use the Errorx, Warnx, Infox, and Debugx methods. Lower perf
```

Comment:
I am ok if you check in this way, and I am not very familiar with zap logging to say how hard using zapcore.Field is in your code,  but I don't think you're setting the right precedent here by giving the most common name to the less performant option.